README.md

```diff
@@ -1,7 +1,43 @@
+# README
+
+### [EN]
+
```

Comment:
Can you add a one-line proposition value?

Reply:
I don't know what you mean